DistroLauncher/DistroLauncher.vcxproj

```
@@ -30,27 +30,27 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>

```

As the automated process to publish the program to the store uses Visual Studio 2022, it is preferable to update the BUILDING.md to use it.

```suggestion
    <PlatformToolset>v143</PlatformToolset>
```

---

Alright. I'll revert these changes in the next push to this PR. 

I'm working on build instructions using VS 2022 and the non-legacy rootfs build repo, but I'm running into unrelated issues with debootstrap failing to successfully install sudo in the rootfs.